deploy/docs/Installation_with_Helm.md

```diff
@@ -36,66 +41,84 @@ These steps require that no Prometheus exists. If you already have Prometheus in
+Next you can run `helm upgrade --install` to install or upgrade our chart.  An example command with the minimum parameters is provided below for Helm2 and Helm3.
 
+Helm2
 ```bash
-helm install sumologic/sumologic --name collection --namespace sumologic --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>"
+helm upgrade --install sumologic/sumologic --name my-release --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>"
 ```
 
-If you get `Error: customresourcedefinitions.apiextensions.k8s.io "alertmanagers.monitoring.coreos.com" already exists`, run the above command with the `--no-crd-hook` flag:
-
+Helm3
 ```bash
-helm install sumologic/sumologic --name collection --namespace sumologic --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>" --no-crd-hook
+helm upgrade --install my-release sumologic/sumologic  --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>"
 ```
 
-If you get `Error: collector with name 'sumologic' does not exist
-sumologic_http_source.default_metrics_source: Importing from ID`, you can safely ignore it and the installation should complete successfully. The installation process creates new [HTTP endpoints](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source) in your Sumo Logic account, that are used to send data to Sumo. This error occurs if the endpoints had already been created by an earlier run of the installation process.
+Once you have completed installation, you can [install the Kubernetes App and view the dashboards](https://help.sumologic.com/07Sumo-Logic-Apps/10Containers_and_Orchestration/Kubernetes/Install_the_Kubernetes_App_and_view_the_Dashboards) or [open a new Explore tab](https://help.sumologic.com/Solutions/Kubernetes_Solution/05Navigate_your_Kubernetes_environment) in Sumo Logic.
 
-__NOTE__ `Google Kubernetes Engine (GKE)` uses Container-Optimized OS (COS) as the default operating system for its worker node pools. COS is a security-enhanced operating system that limits access to certain parts of the underlying OS. Because of this security constraint, Falco cannot insert its kernel module to process events for system calls. However, COS provides the ability to use extended Berkeley Packet Filter (eBPF) to supply the stream of system calls to the Falco engine. eBPF is currently only supported on GKE and COS. For more information see [Installing Falco](https://falco.org/docs/installation/).
+### Troubleshooting Installation
+If you get `Error: customresourcedefinitions.apiextensions.k8s.io "alertmanagers.monitoring.coreos.com" already exists` on a Helm2 installation, run the above command with the `--no-crd-hook` flag:
```

Comment:
the above command uses `helm upgrade --install`

Reply:
Good point, I actually need to test this as I vaguely remember an issue with `--no-crd-hook` not working with the `helm upgrade` command.  Will test that and update here with the findings.

Reply:
there is no `--no-crd-hook` flag for upgrade. I was using `--no-hooks` for upgrades

Reply:
For install (or upgrade) if the CRD hooks do not exist, the solution is to add `--set prometheus-operator.prometheusOperator.createCustomResource=false`.  This works in the `helm upgrade --install` command.  You can't use `-no-hooks` with this command because that prevents the setup job when the chart is not yet installed.  And `--no-crd-hooks` does not work with `upgrade`.

Reply:
I Will update to reflect this.

Reply:
```suggestion
If you get `Error: customresourcedefinitions.apiextensions.k8s.io "alertmanagers.monitoring.coreos.com" already exists` on a Helm2 installation, run the above command with the `--set prometheus-operator.prometheusOperator.createCustomResource=false` parameter:
```